MediaGalleryUi/view/adminhtml/ui_component/standalone_media_gallery_listing.xml

```diff
@@ -211,8 +211,6 @@
                 </item>
             </argument>
             <settings>
-                <label translate="true">Uploaded Date</label>
```

Comment:
Let's keep the label here. Same as the media_gallery_listing.xml.

Reply:
Actually this columns is not visible in the grid and label appears now in filters that located in listingTop

Reply:
We should remove the label from  `media_gallery_listing.xml.` as well

Reply:
Agree, the column isn't visible, but that might be useful later. I would still keep it since it is not related to the bug itself. What do you think @sivaschenko?